flang/test/Driver/mlink-builtin-bc.f90

```diff
@@ -0,0 +1,18 @@
+
+!----------
+! RUN lines
+!----------
+! Embed something that can be easily checked
+! RUN: %flang_fc1 -emit-llvm -triple x86_64-unknown-linux-gnu -o - -mlink-builtin-bitcode %S/Inputs/bclib.bc %s 2>&1 | FileCheck %s
```

Comment:
1. What's bclib.bc and what makes it easy to check?

2. Does the triple matter?

Reply:
The triple matters because when linking the data layout in bclib.bc must match the data layout of the compiled source code, and depending on the host it may not be the same so it is safest to specify the triple to ensure they are the same.